simd/simd-memtest/compiler.h

```diff
@@ -32,6 +34,12 @@
 typedef int64_t __int64;
 #endif
 
+#if defined(__GNUC__) || defined(__INTEL_COMPILER)
```

Comment:
In some places inline ASM was protected by `__INTEL_COMPILER`, but clang and gcc work fine with it (indeed, gcc invented it I guess), so this allows inline ASM to be used wtih those compilers too.